assets/src/edit-story/components/panels/videoPoster.js

```diff
@@ -33,12 +33,16 @@ import { SimplePanel } from './panel';
 import getCommonValue from './utils/getCommonValue';
 
 function VideoPosterPanel({ selectedElements, onSetProperties }) {
-  const featuredMedia = getCommonValue(selectedElements, 'featuredMedia');
-  const poster = getCommonValue(selectedElements, 'poster');
-  const [state, setState] = useState({ featuredMedia, poster });
+  const resource = getCommonValue(selectedElements, 'resource');
```

Comment:
I don't think any `resource` object would ever be identity-equal to any other. `getCommonValue` doesn't work here.

Reply:
Note this is another issue - https://github.com/google/web-stories-wp/issues/572